source/core.py

```diff
@@ -262,6 +262,55 @@ def onEndSession(evt):
 	import windowUtils
 	class MessageWindow(windowUtils.CustomWindow):
 		className = u"wxWindowClassNR"
+		oldBatteryStatus = None

```

Comment:
Any reason you put this on the class and then set it on the class later? It seems to me this would be more appropriate as an instance variable.